lib/matplotlib/tests/test_axes.py

```diff
@@ -2976,7 +2976,7 @@ def test_scatter_singular_plural_arguments(self):
 
 
 def _params(c=None, xsize=2, *, edgecolors=None, **kwargs):
-    return (c, edgecolors, kwargs if kwargs is not None else {}, xsize)
+    return (c, edgecolors, kwargs, xsize)
```

Comment:
`kwargs` defaults to a dict if not provided, so the check here was redundant and raising an error